dotnet-interactive/JupyterFormattingKernelExtensions.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft. All rights reserved. 
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.DotNet.Interactive.Commands;
+using Microsoft.DotNet.Interactive.CSharp;
+using Microsoft.DotNet.Interactive.FSharp;
+using Microsoft.DotNet.Interactive.Jupyter.Formatting;
+
+namespace Microsoft.DotNet.Interactive.App
+{
+    public static class KernelFormattingExtensions
+    {
+        public static IKernel UseJupyterFormatting(this CSharpKernel kernel)
```

Comment:
The return type should be `CSharpKernel`.

Reply:
done